pkg/logs/decoder/auto_multiline_handler.go

```diff
@@ -95,14 +97,15 @@ func NewAutoMultilineHandler(outputChan chan *Message,
 		detectedPattern: detectedPattern,
 	}
 
-	h.singleLineHandler = NewSingleLineHandler(outputChan, lineLimit)
+	h.singleLineHandler = NewSingleLineHandler(inputChan, outputChan, lineLimit)
```

Comment:
This doesn't seem right; `singleLineHandler` used to have a dedicated input channel, but now shares it with multi-line handler. Looks like the code isn't using it as an actor in this case, so it doesn't matter, but maybe `nil` would be more appropriate?

Reply:
Good catch -- I'll look at this more closely!